pkgs/development/libraries/srt/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake, openssl
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "srt-${version}";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Haivision";
+    repo = "srt";
+    rev = "v${version}";
+    sha256 = "0cv73j9c8024p6pg16c4hiryiv4jpgrfj2xhfdaprsikmkdnygmz";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ openssl ];
+
+  configurePhase = "cmake . -DCMAKE_INSTALL_PREFIX=$out";
```

Comment:
```suggestion
  cmakeFlags = [
    # srt incorrectly disables GNUInstallDirs when LIBDIR is specified
    "-UCMAKE_INSTALL_LIBDIR"
  ];
```

Reply:
https://github.com/Haivision/srt/issues/538

Reply:
Great, I applied it in #54380 (which this PR is based upon).